README.md

```diff
@@ -1,10 +1,177 @@
+- **Git and GitHub**: Contributions are made via GitHub. A basic understanding of how to fork a repository, create branches, and submit pull requests is necessary.
+
+**SigLens Platform**: Having a basic understanding of what SigLens offers will help you write better documentation. We encourage you to try out SigLens using 
+
+- The [Siglens Playground](http://playground.sigscalr.io:5122/), or
+- Installing SigLens in your local machine
+
+### &emsp; <a href="https://siglens.github.io/siglens-docs/installation/git" target="_blank">Git</a> &emsp; | &emsp; <a href="https://siglens.github.io/siglens-docs/installation/docker" target="_blank">Docker</a> &emsp;| &emsp; <a href="https://siglens.github.io/siglens-docs/installation/helm" target="_blank">Helm</a>
+
+## Documentation Workflow
+
+SigLens documentation uses [Docusaurus](https://docusaurus.io/docs) for creating and managing [SigLens Documentation](https://siglens.github.io/siglens-docs) website.
+
+To host this documentation on your local machine, follow these steps.
+
+1. Clone the repository
+    ```
+    git clone https://github.com/<username>/siglens-docs.git
+    ```
+2. Install the dependencies
+    ```
+    npm install
+    ```
+3. Start the document website server
+    ```
+    npm start
+    ```
+
+### Markdown structure
+
+Here's an a detailed overview of the entire markdown file structure of the documentation.
 
-### To serve docs locally, run
 ```
-$ npm start
+|-- /docs
+    |-- alerts.md
+|-- /benchmarks
+|   |-- /benchmarks
+|       |-- nyc-taxi-benchmark-runbook.md
+    |-- cluster-health.md
+    |-- community.md
+    |-- contribution-guidelines.md
+    |-- dashboards.md
+|-- /installation
+|   |-- /installation
+|       |-- docker.md
+|       |-- git.md
+|       |-- helm.md
+|-- /instrument-traces
+|   |-- /instrument-traces
+|       |-- dotnet-app.md
+|       |-- go-app.md
+|       |-- java-app.md
+|       |-- js-app.md
+|       |-- python-app.md
+    |-- introduction.md
+    |-- key-concepts.md
+|-- /log-ingestion
+|   |-- /beats
+|   |   |-- /beats
+|   |       |-- filebeat-elasticsearch.md
+|   |       |-- metricbeat-elasticsearch.md
+|   |-- /log-ingestion
+|       |-- elastic-search.md
+|   |-- /fluentd
+|   |   |-- /fluentd
+|   |       |-- fluentd-elasticsearch.md
+|   |       |-- fluentd-splunk.md
+|       |-- fluentd-fluentbit.md
+|   |-- /logstash
+|   |   |-- /logstash
+|   |       |-- logstash-elasticsearch.md
+|   |       |-- logstash-splunk.md
+|       |-- loki.md
+|       |-- open-telemetry.md
+|       |-- splunk-hec.md
+|   |-- /vector
+|   |   |-- /vector
+|   |       |-- vector-elasticsearch.md
+|   |       |-- vector-install.md
+|   |       |-- vector-splunk-hec-logs.md
+|   |       |-- vector-splunk-hec-metrics.md
+|       |-- vector.md
+    |-- log-query-builder.md
+    |-- minion-searches.md
+    |-- retention.md
+    |-- saved-searches.md
+    |-- searching-logs.md
```

Comment:
this flow is going to keep changing as we keep adding, removing sections/sub-sections. Do we want to keep this here ?